addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -159,12 +159,15 @@ def attributes(self, record, field_name, options, values):
 
         if options['translate'] and field.type in ('char', 'text'):
             lang = record.env.lang or 'en_US'
-            if lang == 'en_US':
+            lang_base = 'en_US'
+            if 'website_id' in record._fields:
+                lang_base = record.website_id.default_lang_id.code
+            if lang == lang_base:
                 attrs['data-oe-translation-state'] = 'translated'
             else:
-                value_en = record.with_context(lang='en_US')[field_name]
+                value_base = record.with_context(lang=lang_base)[field_name]
                 value_lang = record.with_context(lang=lang)[field_name]
-                attrs['data-oe-translation-state'] = 'translated' if value_en != value_lang else 'to_translate'
+                attrs['data-oe-translation-state'] = 'translated' if value_base != value_lang else 'to_translate'
```

Comment:
Doesn't this is conceptually incorrect even before this PR?
Does it means that a term which has the same value in its translation (eg a name / brand, a sentence/word used in other languages, a term which has same translation as another language) will always be considered as translated?

I guess you don't have another solution technically that this basic comparison heuristic with jsonb now?

I guess it's not the end of the world for the website part, worst case sentences/words will remain flagged as non translated while they are. It's not the end of the world, we can live with that.
I am not sure how it impacts the rest of Odoo tho -> Are those returned as non translated terms when exporting untranslated terms? Is this export feature still a thing actually?

Reply:
The 'to translate' tag now is only used by website in edit_translation mode
When exporting translations, we alwasy use en_US as the source, so the translation is the same as the source it will be exported like
```
msgid "Odoo"
msgstr ""
```
But it should be OK usually